mapreads/Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM debian:8.6
+FROM maxulysse/samtools:1.1
+# FROM maxulysse/samtools:dev
 
 LABEL author="Maxime Garcia" \
-description="BWA 0.7.8 and SAMTools 1.3 image for use in CAW" \
+description="BWA 0.7.8 and SAMTools 1.4 image for use in CAW" \
```

Comment:
In fact, if we still specify in the config file the containers with the 1.0 tag, we have SAMtools 1.3.
SAMTools 1.4, is on the containers with the 1.1 tag